spec/terminology.md

```diff
@@ -3,17 +3,74 @@
 ::: todo
 :::
 
-[[def: method-specific identifier, MSI]]
-~ The `method-specific-id` part of DID Syntax, as defined in [DID Core](https://www.w3.org/TR/did-core/#did-syntax). See section [Method-Specific Identifier](#method-specific-identifier).
+[[def: authentic chained data container (ACDC), ACDC]]
```

Comment:
Relevant to did:webs?

Reply:
ACDCs are mentioned multiple times, so we might as well put it in the terminology. I can remove it if you think it is unnecessary.